### PR TITLE
DAT-3921 add focusing on inputs in Infobox Builder

### PR DESCRIPTION
**Pull request by dianafa**
## Links

https://wikia-inc.atlassian.net/browse/DAT-3921

## Description
Add focusing on input fields when displaying sidebar.

## Reviewers

@RafLeszczynski

### CODE REVIEW

File: front/main/app/mixins/infobox-builder-edit-item.js [hunk 1]
```diff
@@ -6,7 +6,33 @@ export default Ember.Mixin.create(
...
+		 * Focuses on the end of the first text input element of this component
+		 * @returns {void}
+		 */
+		focusFirstInput() {
+			const input = this.$('.text-field-input');
+			let firstInput;
+
+			if (input) {
+				firstInput = input.first().get(0);
+				input.first().focus();
```
Comment (RafLeszczynski) [inline front/main/app/mixins/infobox-builder-edit-item.js]: Please consider avoiding duplication DOM traversal. Consider caching `input.first()` in local variable.

**Comment by RafLeszczynski** [comment]
One small optimisation proposal. Except that LGTM

File: front/main/app/mixins/infobox-builder-edit-item.js [hunk 2]
```diff
@@ -6,7 +6,35 @@ export default Ember.Mixin.create(
...
+		/**
+		 * Focuses on the end of the first text input element of this component
+		 * @returns {void}
+		 */
+		focusFirstInput() {
+			const inputs = this.$('.text-field-input');
+			let firstInput,
+				inputProperties;
+
+			if (inputs) {
```
Comment (bognix) [inline front/main/app/mixins/infobox-builder-edit-item.js]: It will always be truthy - check in console: `!!$('not-existing-tag')`

**Comment by bognix** [comment]
two minor things - when fixed - LGTM

File: front/main/app/mixins/infobox-builder-edit-item.js (hunk 2 above)
Comment (RafLeszczynski) [inline front/main/app/mixins/infobox-builder-edit-item.js]: that's true cause you will always receive jQuery collection object. If there won't be any elemenets with class name `.text-field-input` this collection will have `length = 0`

Comment (RafLeszczynski) [inline front/main/app/mixins/infobox-builder-edit-item.js]: we did have an issue with ember liquid fire (used here before, now removed) that `didRender` hook was fired twice and in second call `this.$('.text-field-input');` returns `undefined`

**Comment by RafLeszczynski** [comment]
LGTM if build passes

**Pull request merged**